### PR TITLE
Added tutorial repeat voice command, modified Jenna's text, added IsL…

### PR DESCRIPTION
**Pull request by marjasi**
…istening property to Speech Recognizer.

### CODE REVIEW

**Comment by marjasi** [comment]
[[Feature] Implement tutorial voice command.](https://trello.com/c/QksQ59zf/132-feature-implement-tutorial-voice-command)

File: Mobile/ShopLens/Droid/Activities/MainActivity.cs [hunk 1]
```diff
@@ -87,14 +89,14 @@ protected override void OnCreate(Bundle savedInstanceState)
 
             if (!talkBackEnabled)
             {
-               InitiateNoTalkBackMode();
+                InitiateNoTalkBackMode();
```
Comment (EdvardasDlugauskas) [inline Mobile/ShopLens/Droid/Activities/MainActivity.cs:92]: Not sure if fixing bad indentation...
image
... or adding bad indentation

File: Mobile/ShopLens/Droid/Helpers/ShopLensSpeechRecognizer.cs [hunk 2]
```diff
@@ -6,26 +6,37 @@ namespace ShopLens.Droid.Helpers
 {
     public class ShopLensSpeechRecognizer : ISpeechRecognizer
     {
+        public bool IsListening
+        {
+            get { return voiceListener != null ? true : false; }
```
Comment (EdvardasDlugauskas) [inline Mobile/ShopLens/Droid/Helpers/ShopLensSpeechRecognizer.cs]: :thinking: :thinking: :thinking: 

:rofl: 

This isn't a rare thing to see, but it's really funny when you think about it

I'm sorry, but I'll leave this one for you to figure out :wink: 

Hint: `voiceListener != null ? true : false`

**Review by EdvardasDlugauskas — COMMENTED** [review]
Stellar job m:eight::zero: 

But

So... Many... Formatting changes :face_with_head_bandage:

File: Mobile/ShopLens/Droid/Activities/MainActivity.cs (hunk 1 above)
Comment (marjasi) [reply Mobile/ShopLens/Droid/Activities/MainActivity.cs:92]: I swear to god, I didn't do it on purpose.

File: Mobile/ShopLens/Droid/Helpers/ShopLensSpeechRecognizer.cs (hunk 2 above)
Comment (marjasi) [reply Mobile/ShopLens/Droid/Helpers/ShopLensSpeechRecognizer.cs]: Oh, I could just do "return voiceListener != null", I see.

File: Mobile/ShopLens/Droid/Activities/MainActivity.cs (hunk 1 above)
Comment (marjasi) [reply Mobile/ShopLens/Droid/Activities/MainActivity.cs:92]: It was Ctrl + K + D, from now on I won't select the whole document text with it.

Comment (marjasi) [reply Mobile/ShopLens/Droid/Activities/MainActivity.cs:92]: I mean the other formatting changes were made by it.

**Pull request merged**